lib/sisimai/bite/email/messagelabs.rb

```diff
@@ -0,0 +1,213 @@
+module Sisimai::Bite::Email
+  module MessageLabs
+    # Sisimai::Bite::Email::MessageLabs parses a bounce email which created by
+    # Symantec.cloud: formerly MessageLabs. Methods in the module are called
+    # from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/MessageLabs.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->